### PR TITLE
New weapon(?) - Medi-Beam

### PR DESCRIPTION
**Pull request by Stop-Signs**
## About the PR
Healing beam GOOOO. A new t3 experimental tech that provides both sec and med with a healing cannon!

## Why / Balance
Ranged healing is cool, the syringe gun already helps a little with that, hopefully this will also be useful. It heals as much as a bruise pack, slightly less than an ointment, and about 1.5u of epi worth of airloss, per shot.

## To Do List

- [x] figure out why its crashing
- [x] fix the sprites
- [x] add the research
- [x] add the recipe
- [x] fix any errors with that
## Requirements

- [x] I have tested all added content and changes.
- [x] I have added media to this PR or it does not require an ingame showcase.

https://github.com/user-attachments/assets/ea89acc6-58fd-4d19-aa00-94b47a680b14

## Breaking changes

**Changelog**

:cl:
- add: Epistemics can now research a ranged healing cannon

### CODE REVIEW

**Comment by CybersunOperative** [comment]
Hi, Hello. Direction here.
I have a few requests to make. Lower the firerate just a smidge from 3 to 2. The description also does not seem fitting (IK-60 item desc) so I recommend you come up with something else fitting for the weapon. I also ask that you add one bar of gold to the crafting recipe, to make it somewhat harder to make and moves it's recipe more in line with other T3s. Besides that, seems fine. Minor amounts of healing, decent ammo capacity. Good support weapon or medical auxiliary.

**Comment by Stop-Signs** [comment]
> Hi, Hello. Direction here. I have a few requests to make. Lower the firerate just a smidge from 3 to 2. The description also does not seem fitting (IK-60 item desc) so I recommend you come up with something else fitting for the weapon. I also ask that you add one bar of gold to the crafting recipe, to make it somewhat harder to make and moves it's recipe more in line with other T3s. Besides that, seems fine. Minor amounts of healing, decent ammo capacity. Good support weapon or medical auxiliary.

all requested changes have been addressed

File: Resources/Locale/en-US/research/technologies.ftl [hunk 1]
```diff
@@ -52,6 +52,7 @@ research-technology-advanced-anomaly-research = Advanced Anomaly Research
 research-technology-rped = Rapid Part Exchange
 research-technology-super-parts = Super Parts
 research-technology-deterrence = Deterrence Technologies
+research-technology-biopulse-emitter = Bio-Pulse Emitter
```
Comment (MilonPL) [inline Resources/Locale/en-US/research/technologies.ftl]: namespace

File: Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml [hunk 2]
```diff
@@ -261,3 +261,40 @@
     steps: 5
     zeroVisible: true
   - type: Appearance
+
+- type: entity
+  name: Medi-Cannon
+  parent: BaseWeaponBattery
+  id: WeaponHealGun
+  description: "A high-tech firearm that shoots specialized healing projectiles."
+  components:
```
Comment (MilonPL) [inline Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml]: proto order

File: Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+ - type: entity
+   name: heal bolt
+   id: BulletHealLaser
+   parent: BaseBullet
+   categories: [ HideSpawnMenu ]
```
Comment (MilonPL) [inline Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml]: entity prototype order

File: Resources/Prototypes/DeltaV/Recipes/Lathes/security.yml [hunk 4]
```diff
@@ -223,4 +223,3 @@
   materials:
     Steel: 1000
     Plastic: 800
-
```
Comment (MilonPL) [inline Resources/Prototypes/DeltaV/Recipes/Lathes/security.yml]: why

**Review by MilonPL — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/DeltaV/Recipes/Lathes/security.yml (hunk 4 above)
Comment (Stop-Signs) [reply Resources/Prototypes/DeltaV/Recipes/Lathes/security.yml]: On master there is a random line 227, which I deleted. I believe that's why it looks wonky. do I need to untouch this?

Comment (MilonPL) [reply Resources/Prototypes/DeltaV/Recipes/Lathes/security.yml]: yes why would you delete it

File: Resources/Prototypes/DeltaV/Recipes/Lathes/medical.yml [hunk 5]
```diff
@@ -22,3 +22,13 @@
   materials:
     Plastic: 150
     Steel: 50
+
+- type: latheRecipe
+  id: WeaponHealGun
+  result: WeaponHealGun
+  completetime: 1.5
+  materials:
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Recipes/Lathes/medical.yml]: parent it to BaseWeaponRecipe so it cant be made instantly (remove completeTime: 1.5) and is in weapons category

File: Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml [hunk 6]
```diff
@@ -261,3 +261,40 @@
     steps: 5
     zeroVisible: true
   - type: Appearance
+
+- type: entity
+  parent: BaseWeaponBattery
+  id: WeaponHealGun
+  name: Medi-Cannon
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml]: should call it medibeam and if not medi-cannon for consistence with everything else

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Comment by Stop-Signs** [comment]
I changed the color to be a mintier (lighter) green

**Review by Lyndomen — CHANGES_REQUESTED** [review]
Meowdy, Direction has voted to **Deny** this PR, as we feel there is too much overlap with the existing Syringe Gun, and also do not want to "speed up" the process of healing people. We believe we would like to revisit this concept after woundmed is real (or if forever fake), as it would likely need to be completely changed then either way.

**Comment by Stop-Signs** [comment]
> Meowdy, Direction has voted to **Deny** this PR, as we feel there is too much overlap with the existing Syringe Gun, and also do not want to "speed up" the process of healing people. We believe we would like to revisit this concept after woundmed is real (or if forever fake), as it would likely need to be completely changed then either way.

😭 (im gonna recycle the sprite for a caustic blaster 😉

**Pull request closed without merging**